lib/model.js

```diff
@@ -2676,6 +2680,9 @@ Model.findByIdAndUpdate = function(id, update, options, callback) {
       throw new TypeError(msg);
     }
     return this.findOneAndUpdate({ _id: id }, undefined);
+  } else if (id === undefined) {
+    const msg = 'Model.findByIdAndUpdate(): id cannot be undefined if update is specified.\n';
```

Comment:
Why `\n`?

Reply:
Oh, I was just following the format of the preceding error messages. I went ahead and removed it